src/node_crypto.cc

```diff
@@ -2785,8 +2785,12 @@ inline bool CertIsStartComOrWoSign(X509_NAME* name) {
     startcom_wosign_data = dn.data;
     startcom_wosign_name = d2i_X509_NAME(nullptr, &startcom_wosign_data,
                                          dn.len);
-    if (X509_NAME_cmp(name, startcom_wosign_name) == 0)
```

Comment:
Should we just store the result to `int cmp` and free before the condition test?